frontend/src/metabase/components/ListItem/ListItem.jsx

```diff
@@ -8,44 +8,42 @@ import Card from "metabase/components/Card";
 import S from "metabase/components/List/List.css";
 import Icon from "metabase/components/Icon";
 
-//TODO: extend this to support functionality required for questions
-const ListItem = ({ index, name, description, placeholder, url, icon }) => (
-  <Link to={url} className="text-brand-hover">
-    <Card hoverable className="mb2 p3 bg-white rounded bordered">
-      <div className={cx(S.item)}>
-        <div className={S.itemIcons}>
-          {icon && <Icon className={S.chartIcon} name={icon} size={16} />}
-        </div>
-        <div className={S.itemBody}>
-          <div className={S.itemTitle}>
-            <Ellipsified
-              className={S.itemName}
-              tooltip={name}
-              tooltipMaxWidth="100%"
-            >
-              <h3>{name}</h3>
-            </Ellipsified>
+const ListItem = ({ name, description, placeholder, url, icon }) => (
+  <li className="relative">
+    <Link to={url} className="text-brand-hover">
+      <Card hoverable className="mb2 p3 bg-white rounded bordered">
+        <div className={cx(S.item)}>
+          <div className={S.itemIcons}>
+            {icon && <Icon className={S.chartIcon} name={icon} size={16} />}
           </div>
-          {(description || placeholder) && (
-            <div className={cx(S.itemSubtitle)}>
-              {description || placeholder}
+          <div className={S.itemBody}>
+            <div className={S.itemTitle}>
+              <Ellipsified
+                className={S.itemName}
+                tooltip={name}
+                tooltipMaxWidth="100%"
+              >
+                <h3>{name}</h3>
```

Comment:
This title doesn't get ellipsified because it is a block level element. Will file a separate issue for that.